workflows/scRNAseq/fastq-to-matrix-10x/scrna-seq-fastq-to-matrix-10x-cellplex-tests.yml

```diff
@@ -0,0 +1,112 @@
+- doc: Test outline for scrna-seq-fastq-to-matrix-10X-cellplex.ga
+  job:
+    fastq PE collection GEX:
+      class: Collection
+      collection_type: list:paired
+      elements:
+      - class: Collection
+        type: paired
+        identifier: subsample
+        elements:
+        - identifier: forward
+          class: File
+          path: test-data/GEX_R1.fastqsanger.gz
+          filetype: fastqsanger.gz
+        - identifier: reverse
+          class: File
+          path: test-data/GEX_R2.fastqsanger.gz
+          filetype: fastqsanger.gz
+    reference genome: dm6 # To test on usegalaxy.eu dm6full
+    Barcode Size is same size of the Read: false
+    gtf:
+      class: File
+      location: https://zenodo.org/record/6457007/files/Drosophila_melanogaster.BDGP6.32.109_UCSC.gtf.gz
+      decompress: true
```

Comment:
Is that because we don't have a gtf.gz datatype ? Should we add one ?

Reply:
I think most of tools do not accept gtf.gz so I don't know if it worth to create a datatype?

Reply:
(I just gz before putting to zenodo because it save space)

Reply:
You mean the upstream tools or the galaxy tool wrappers ? In any case this is fine for now.

Reply:
I mean the upstream tools.